src/framework/xr/xr-manager.js

```diff
@@ -696,8 +696,7 @@ class XrManager extends EventHandler {
 
         if (lengthNew > this.views.length) {
             // add new views into list
-            const viewAddCount = (lengthNew - this.views.length);
-            for (let i = 0; i <= viewAddCount; i++) {
+            while (lengthNew > this.views.length) {
```

Comment:
now get rid of that if, and also else?
the code should just be
```
while (..) {
}

whilte (..) {
}
```
no need for ifs as they just do the same as the first iteration of the loop.
```

Reply:
Done!